taxi/views.py

```diff
@@ -34,6 +36,23 @@ class ManufacturerListView(LoginRequiredMixin, generic.ListView):
     paginate_by = 5
 
 
+class ManufacturerCreateView(CreateView):
```

Comment:
Consider adding `LoginRequiredMixin` to `ManufacturerCreateView` to ensure that only authenticated users can create a new manufacturer.